### PR TITLE
feat(AniMi Club): added presence

### PR DESCRIPTION
**Pull request by riktikdev**
## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>
https://i.imgur.com/HnGyqR6
https://i.imgur.com/54QKVEA

</details>

### CODE REVIEW

File: websites/A/AniMi Club/metadata.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	"tags": [
+		"anime",
+		"аниме",
+		"animi",
+		"аними",
+		"animiclub",
+		"анимиклаб"
+	],
+	"iframe": true,
+	"iFrameRegExp": ".*",
```
Comment (theusaf) [inline websites/A/AniMi Club/metadata.json:32]: If possible, set the iframe regex to the urls that it accesses

**Review by theusaf — CHANGES_REQUESTED** [review]
Other than this, looks good.

File: websites/A/AniMi Club/metadata.json (hunk 1 above)
Comment (riktikdev) [reply websites/A/AniMi Club/metadata.json:32]: I'm a little confused about what you're talking about 😅

Comment (theusaf) [reply websites/A/AniMi Club/metadata.json:32]: The current value `.*` is alright, but means that it runs on any iframes on the page. We'd prefer it if the regex only matches the iframe urls that the page actually uses for its video features.

For example, if the page uses video sources from `example.com` and `foo.com`, a potential regex could be:
```json
"(example[.]com|foo[.]com)[/]"
```

Comment (riktikdev) [reply websites/A/AniMi Club/metadata.json:32]: Unfortunately there are links in the iframe that I may not be aware of. The API I use for the player may give different links.

**Review by theusaf — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**